xml/System/SequencePosition.xml

```diff
@@ -29,7 +29,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Represents position in a non-contiguous set of memory. Properties of this type shouldn't be interpreted by anything but the type that created it.</summary>
```

Comment:
nit: 
Maybe make this a remark instead:
`Properties of this type shouldn't be interpreted by anything but the type that created it.`

Reply:
I will submit your suggestions as a new review because this one has already been merged.